### PR TITLE
fix: Sell item wrong URL

### LINKED ISSUE #145: Sell item wrong URL
When clicking sell CTA on any collectible item, it took the players to their assets URL on the marketplace when it should be taking them directly to the wearable URL instead.

### PR DESCRIPTION
**Pull request by sandrade-dcl**
## What does this PR change?

Fix #145 

When clicking sell CTA on any collectible item, it took the players to their assets URL on the marketplace when it should be taking them directly to the wearable URL instead.

Currently, the sell button takes you to `https://market.decentraland.org/account` and it needs to go to
`https://market.decentraland.org/contracts/[COLLECTION-SMART-CONTRACT]/tokens/[WEARABLE-TOKEN-ID]`

## How to test the changes?

1. Go to: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:fix/Sell-item-wrong-URL&ENV=org
2. Open the AvatarEditorHUD.
3. Go to Collectibles tab.
4. Click on the `i` icon of any wearable.
5. Click on the SELL button.
6. Notice that you will be redirected to the correct wearable page.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs [hunk 1]
```diff
@@ -127,6 +131,32 @@ private void LoadOwnedWereables(UserProfile userProfile)
...
+
+        DCL.Environment.i.platform.serviceProviders.theGraph.QueryNftCollections(userProfile.userId, NftCollectionsLayer.ETHEREUM)
+           .Then((nfts) =>
+           {
+               ownedNftCollectionsL1 = nfts;
+           })
+           .Catch((error) =>
+           {
+               Debug.LogError(error);
+           });
```
Comment (AjimenezDCL) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs:147]: I think you can compact this without sacrificing much readability.
```suggestion
        DCL.Environment.i.platform.serviceProviders.theGraph.QueryNftCollections(userProfile.userId, NftCollectionsLayer.ETHEREUM).Then(nfts => ownedNftCollectionsL1 = nfts).Catch(Debug.LogError);
```

**Review by AjimenezDCL — APPROVED** [review]

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs (hunk 1 above)
Comment (AjimenezDCL) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs:147]: Haven't test the build, but codewise looks good! Left a small suggestion

**Comment by AjimenezDCL** [comment]
Haven't test the build, but codewise looks good! Left a small suggestion

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs (hunk 1 above)
Comment (AjimenezDCL) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs:147]: You can apply the same to the L2 promise

Comment (sandrade-dcl) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs]: Done in https://github.com/decentraland/unity-renderer/pull/829/commits/88866dbc3429a55ae28e9702446d8001ec5b5918. Thanks Alex!

**Review by kuruk-mm — APPROVED** [review]
It works perfectly!

**Pull request merged**